lib/data-builder.js

```diff
@@ -67,8 +67,10 @@ function fillDebugData(data, err) {
     if ((p in data)) continue;
     data[p] = err[p];
   }
-  // NOTE err.stack is not an enumerable property
+  // NOTE `err.stack` and `err.message` is not an enumerable property
   data.stack = err.stack;
+  if (err.message)
```

Comment:
I think we need to handle this edge case in fillBadRequestError too, shouldn't we?

Another option, which may be better, is to change the code handling non-errors so that it modified the `err` object instead of the `data` object:

```js
 if (typeof err !== 'object') {
   err = {
      statusCode: 500,
      message: '' + err,
  }
}
```

And then we can do `data.message = err.message` everywhere else.

@davidcheung Thoughts?